dvc/command/experiments.py

```diff
@@ -601,6 +601,7 @@ def run(self):
 
 def add_parser(subparsers, parent_parser):
     EXPERIMENTS_HELP = "Commands to run and compare experiments."
+    LAST_CHECKPOINT = ":last"
```

Comment:
@pmrowla, above import `dvc.repo.experiment` was importing git-related stuffs, dvcfiles and stages which are slow. Instead of refactoring there, I choose to keep it simple by duplicating.